cli/bpmetadata/schema/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+
+	log "github.com/inconshreveable/log15"
+)
+
+// bpmetadata log15 handler
+var Log = log.New()
+
+func main() {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr,
+			"USAGE: %s [-output=PATH]\n",
+			path.Base(os.Args[0]))
+		flag.PrintDefaults()
+		os.Exit(1)
+	}
+
+	output := flag.String("output", "", "output path for generating the JSON schema definition")
```

Comment:
Since we are not planning to expose this for users, is there value in making the output configurable via flags?

Reply:
I think there is because we may need to change the location at some point and it's easier if there is already a flag available. There's not much overhead, let's just leave it.